src/layouts/ResourceCategory/ResourceCategory.tsx

```diff
@@ -0,0 +1,108 @@
+import { Box, Text, Skeleton, SimpleGrid } from "@chakra-ui/react"
+import { BiBulb } from "react-icons/bi"
+import {
+  Switch,
+  useRouteMatch,
+  useHistory,
+  Link as RouterLink,
+} from "react-router-dom"
+
+import { useGetDirectoryHook } from "hooks/directoryHooks"
+
+// Import screens
+import {
+  CreateButton,
+  Section,
+  SectionCaption,
+  SectionHeader,
+} from "layouts/components"
+import { SiteViewLayout } from "layouts/layouts"
+import {
+  PageSettingsScreen,
+  MoveScreen,
+  DeleteWarningScreen,
+} from "layouts/screens"
+import { isResourcePageData } from "layouts/utils"
+
+import { ProtectedRouteWithProps } from "routing/RouteSelector"
+
+// Import utils
+import { ResourceCategoryRouteParams } from "types/resources"
+import { deslugifyDirectory } from "utils"
+
+import { ResourceCard, ResourceCategoryBreadcrumb } from "./components"
+
+export const ResourceCategory = (): JSX.Element => {
+  const { params } = useRouteMatch<ResourceCategoryRouteParams>()
+  const { resourceCategoryName: collectionName } = params
+  const { path, url } = useRouteMatch()
+  const history = useHistory()
+
+  const { data: _pagesData, isLoading } = useGetDirectoryHook({
+    ...params,
+  })
+
+  const pagesData =
+    _pagesData && _pagesData.length > 0
+      ? (_pagesData as unknown[]).filter(isResourcePageData)
+      : []
+
+  return (
+    <>
+      <SiteViewLayout>
+        <Section>
+          <Box>
+            <Text as="h2" textStyle="h2">
+              {deslugifyDirectory(collectionName)}
+            </Text>
+            <ResourceCategoryBreadcrumb />
+          </Box>
+        </Section>
+        <Section>
+          <Box w="full">
+            <SectionHeader label="Pages">
+              <CreateButton as={RouterLink} to={`${url}/createPage`}>
+                Create Page
+              </CreateButton>
+            </SectionHeader>
+            <SectionCaption icon={BiBulb} label="PRO TIP: ">
+              Organise your workspace by moving pages into folders
+            </SectionCaption>
+          </Box>
+          <Skeleton isLoaded={!isLoading}>
+            <SimpleGrid columns={3} spacing="1.5rem">
+              {/* NOTE: need to use multiline cards */}
+              {pagesData
+                .filter(
+                  (page: { name: string }) => page.name !== "contact-us.md"
+                )
+                .map(({ name, date }) => (
+                  <ResourceCard title={name} date={date} />
+                ))}
```

Comment:
just double checking here - the additional utlity which `retrieveResourceFileMetadata` provides is that there's conversion of whitespace to hyphens in the title?

Reply:
It splits the file name ({date}-{type}-{title}) into the respective components, so you would be able to retrieve the relevant fields more easily

Reply:
i didn't use this as the original format from the be is already split into the relevant fields!

Reply:
Ah got it - it might still be helpful to get the truncated version of the title though, as opposed to `{date}-{type}-{title}` which is currently being displayed

Reply:
As discussed offline - title should be `title` (which refers to the title in the front matter) and not `name` (which refers to the entire file name `{date}-{type}-{title}`) - `retrieveResourceFileMetadata` is not necessary